### PR TITLE
Add pl.youkai.nscan

### PR DESCRIPTION
**Pull request by youkai-pl**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by youkai-pl** [comment]
bot, build pl.youkai.nscan

**Comment by youkai-pl** [comment]
bot, build pl.youkai.nscan

File: pl.youkai.nscan.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+app-id: pl.youkai.nscan
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.golang
+command: nscan -x
```
Comment (hfiguiere) [inline pl.youkai.nscan.yml:7]: command is NOT a command line.

Comment (hfiguiere) [reply pl.youkai.nscan.yml:7]: That's the build failure:

```
Error: Command 'nscan -x' not found. Use a shell wrapper for passing arguments
```

**Comment by youkai-pl** [comment]
bot, build pl.youkai.nscan

**Comment by youkai-pl** [comment]
bot, build pl.youkai.nscan

**Pull request closed without merging**